docs/ztp-precaching.md

```diff
@@ -0,0 +1,302 @@
+
+```
+installerArgs: '["--save-partlabel", "data"]'
+```
+
+Several extra options can be passed to the coreos-installer utility. Here below, you can see the most interesting ones:
+
+```
+# coreos-installer install --help
+coreos-installer-install 0.12.0
+Install Fedora CoreOS or RHEL CoreOS
+
+USAGE:
+    coreos-installer install [OPTIONS] <dest-device>
+
+OPTIONS:
+...
+    -u, --image-url <URL>           
+            Manually specify the image URL
+
+    -f, --image-file <path>         
+            Manually specify a local image file
+
+    -i, --ignition-file <path>      
+            Embed an Ignition config from a file
+
+    -I, --ignition-url <URL>        
+            Embed an Ignition config from a URL
+...            
+        --save-partlabel <lx>...    
+            Save partitions with this label glob
+
+        --save-partindex <id>...    
+            Save partitions with this number or range
+...
+        --insecure-ignition         
+            Allow Ignition URL without HTTPS or hash
+...
+ARGS:
+    <dest-device>    
+            Destination device
+```
+
+###  4.3. <a name='nodes.ignitionConfigOverride'></a>nodes.ignitionConfigOverride
+
+The purpose of this field is very similar to [clusters.ignitionConfigOverride](./ztp-config.md#clustersignitionconfigoverride). Unlike the previous ignitionOverride, we are in the OCP installation stage. This means that we need to extract and load the OCP images that are needed to install the cluster. Remember that in the discovery stage we are only taking care of the container images required for discovery. 
+
+> :warning: The number of container images extracted and loaded is way bigger than in the discovery stage. So, depending on the OCP release and whether it was requested to install the telco operators, the time it takes will vary. 
+
+* There are two systemd units (var-mnt.mount and precache-ocp.services). The precache-ocp.service depends on the disk partition to be mounted in /var/mnt by the var-mnt unit. The precache-ocp service basically calls a script called `extract-ocp.sh`. Notice that the precache-ocp must be executed before the machine-config-daemon-pull.service and nodeip-configuration.service, so it means that extracting the images is done before the OCP installation starts.
+* The `extract-ocp.sh` script basically uncompresses and loads the images required in this stage from the disk partition to the local container storage. Fundamentally there are OCP release images and operators if they were requested to be installed. Once done, images can be used locally instead of pulled down from a registry.
```

Comment:
Are the decoded extraction scripts stored somewhere? Maybe they can be added into the repo with a reference here, noting that they're encoded in this config?